K2Bridge.Tests.UnitTests/Visitors/AggregationVisitorTests.cs

```diff
@@ -26,7 +26,8 @@ public string AggregationVisit_WithEmptyAgg_ReturnsNoPrimary()
         [TestCase(ExpectedResult = "['A']=avg(['fieldA'])")]
         public string AggregationVisit_WithAvgAgg_ReturnsAvgPrimary()
         {
-            var aggregateClause = new AggregationContainer() {
+            var aggregateClause = new AggregationContainer()
```

Comment:
This file just has formatting changes, please reset it and we'll have it in another PR

Reply:
Ok

Reply:
Fixed